Hw18 webflux

daria-grebneva opened the pull request.

daria-grebneva committed:
Добавила основу чата из лекции для выполнения ДЗ

daria-grebneva committed:
Доработала комнату 1408:
 - Убрала возможность отправлять сообщения из этой комнаты
 - Добавила возможность получать все сообщения из других комнат

petrelevich commented on hw18-webflux/client-service/src/main/java/ru/otus/controllers/MessageController.java:
```diff
@@ -0,0 +1,97 @@
+    public void getMessage(@DestinationVariable String roomId, Message message) {
+        logger.info("get message:{}, roomId:{}", message, roomId);
+
+        if (ROOM_FOR_ALL_MESSAGES.equals(roomId)) {
```
datastore ничего не должен знать про "комнату". client-service должен прислать запрос "все сообщения"

daria-grebneva replied:
Сделано!

daria-grebneva committed:
Доработала комнату 1408:
  - datastore теперь не знает про комнату, запрос с клиента идет на отдельный эндпойнт
